lib/oc-tests/test_shareFile.py

```diff
@@ -176,7 +176,7 @@ def shareeOne(step):
     list_files(d)
 
     shared = reflection.getSharedObject()
-    if sharePermissions == (OCS_PERMISSION_READ | OCS_PERMISSION_SHARE):
+    if not sharePermissions & OCS_PERMISSION_UPDATE:
       expect_not_modified(os.path.join(d,'TEST_FILE_MODIFIED_USER_SHARE.dat'), shared['md5_sharer'])
```

Comment:
1. user 1 creates a file
* user 1 shares file with user 2 (permissions read + share)
* user 2 modifies the file locally
* user 2 synces

Should the local file be killed, just because user 2 is not allowed to modify the file upstream? (okay, while writing this it sounds more like a question for the client guys)